packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -22,29 +22,34 @@ const { prefix } = settings;
  */
 const _types = {
   '1-3': [`${prefix}--col-lg-4`, `${prefix}--col-lg-12`], // 1/4 - 3/4
+  '2-3': [`${prefix}--col-lg-8`, `${prefix}--col-lg-4`], // 2/3 - 1/3
```

Comment:
@photodow @RaphaelAmadeu would `2-1` be a more appropriate type?

Reply:
We need a better way to name these the type names here aren't clear to me... `4-12`, `2-1` maybe? It also doesn't account for any differences in responsiveness.

Reply:
I agree, I've just changed the naming on the latest commit